AUTHORS

```diff
@@ -59,3 +59,4 @@ Gaurav Kumar
 Simon Westphahl
 Pedro Ferreira
 Danilo Resende
+wklken
```

Comment:
Only real names should be listed. Github handles can be extracted from git logs.